docs/tutorial.rst

```diff
@@ -336,15 +337,12 @@ give the basic statistics of the model and should look like this:
 .. code-block:: shell
 
     ...
-    WARNING: Species pyr_b was converted to boundary condition because of "_b" suffix
-    WARNING: Species succ_b was converted to boundary condition because of "_b" suffix
-    INFO: Detected biomass reaction: Biomass_Ecoli_core_w_GAM
-    Model: Ecoli_core_model
-    - Biomass reaction: Biomass_Ecoli_core_w_GAM
-    - Compounds: 72
+    Model: Escerichia coli Textbook (core) model (Excel format), Orth et al., 2010
+    - Biomass reaction: None
+    - Compounds: 54
     - Reactions: 95
     - Genes: 137
-    INFO: Using default flux limit of 1000.0
+    INFO: e is extracellular compartment
```

Comment:
You are running this with excel importer even though the command above is for sbml. You need to run this with the command above to see if the output should be changed.